packages/cli/templates/csharp/graph/{{name}}.hbs/{{name}}.csproj.hbs

```diff
@@ -22,7 +22,6 @@
     <PackageReference Include="Microsoft.Teams.Common" Version="2.0.0-preview.*" />
     <PackageReference Include="Microsoft.Teams.Extensions.Hosting" Version="2.0.0-preview.*" />
     <PackageReference Include="Microsoft.Teams.Plugins.AspNetCore" Version="2.0.0-preview.*" />
-    <PackageReference Include="Microsoft.Teams.Plugins.AspNetCore.DevTools" Version="2.0.0-preview.*" />
```

Comment:
devtools should be included/enabled for all the templates https://github.com/microsoft/teams.ts/blob/main/packages/cli/templates/typescript/graph/src/index.ts#L7